### PR TITLE
Fixes #69 - Cloud Run service w/Native Java Image Support

### LINKED ISSUE #69: CodeLab #1 - Add Cloud Run service w/Native Java Image Support

### PR DESCRIPTION
**Pull request by ddobrin**

### CODE REVIEW

**Review by meteatamel — COMMENTED** [review]
Can you put all the code under "java"? This is in case, we want to write the same sample for other languages.

File: services/image-analysis/.gitignore [hunk 1]
```diff
@@ -0,0 +1,33 @@
+HELP.md
```
Comment (meteatamel) [inline services/image-analysis/.gitignore:1]: We already have a .gitignore file at the top of the repository. Could you remove this one and add the Java related ignores in that file?

Comment (meteatamel) [reply services/image-analysis/.gitignore:1]: Also, please add .DS_Store to .gitignore too and remove them from the checkin

File: services/image-analysis/src/main/java/services/EventController.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
+package services;
```
Comment (meteatamel) [inline services/image-analysis/src/main/java/services/EventController.java:1]: We need Google copyright headers in all source files

File: services/image-analysis/src/main/java/services/EventController.java [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
```
Comment (meteatamel) [inline services/image-analysis/src/main/java/services/EventController.java:2]: Copyright header is missing

File: services/image-analysis/README.md [hunk 4]
```diff
@@ -0,0 +1,241 @@
+# Image-analysis Service - Cloud Run Service using Native Spring w/GraalVM
+
+## Build code and publish images
```
Comment (meteatamel) [inline services/image-analysis/README.md:3]: What environment are you assuming? We usually write instructions for Cloud Shell, so anyone can run them without having to setup an environment

**Review by meteatamel — CHANGES_REQUESTED** [review]
Please see my comments

File: services/image-analysis/src/main/java/services/EventController.java (hunk 2 above)
Comment (ddobrin) [reply services/image-analysis/src/main/java/services/EventController.java]: All done

**Pull request merged**